l1-contracts/contracts/bridgehub/MessageRoot.sol

```diff
@@ -157,15 +157,15 @@ contract MessageRoot is IMessageRoot, ReentrancyGuard {
 
     // It is expected that the root is present
     // `_updateTree` should be false only if the caller ensures that it is followed by updating the entire tree.
-    function _unsafeResetChainRoot(uint256 _index, bool _updateTree) internal {
-        uint256 chainId = chainIndexToId[_index];
-        bytes32 initialRoot = chainTree[chainId].setup(CHAIN_TREE_EMPTY_ENTRY_HASH);
-
-        if (_updateTree) {
-            // slither-disable-next-line unused-return
-            sharedTree.updateLeaf(_index, Messaging.chainIdLeafHash(initialRoot, chainId));
-        }
-    }
+    // function _unsafeResetChainRoot(uint256 _index, bool _updateTree) internal {
```

Comment:
?

Reply:
its not used anywhere yet and i was receiving some linting errors